arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java

```diff
@@ -59,11 +60,24 @@ public String getServiceOperation() {
         return getPath() + "." + getOperationName();
     }
     public String getRequest() {
-        Object request = null;
-        if (invocation.getArguments() != null && invocation.getArguments().length > 0) {
-            request = invocation.getArguments()[0];
+        return parseRequest(invocation.getArguments(), Serializer::serialize);
+    }
+    public String getRequestParamType() {
+        Class<?>[] parameterTypes = invocation.getParameterTypes();
+        if (parameterTypes != null && parameterTypes.length > 0) {
+            return parameterTypes[0].getName();
+        }
+        return parseRequest(invocation.getArguments(), TypeUtil::getName);
+    }
+    public static String parseRequest(Object request, Function<Object, String> parser) {
```

Comment:
>  public static String parseRequest

Would you change "public" to internal access level?

Reply:
DubboStreamAdapter#getRequest also use